src/Features/Core/Portable/Completion/CompletionItemRules.cs

```diff
@@ -51,5 +51,14 @@ internal partial class CompletionItemRules
         {
             return null;
         }
+
+        /// <summary>
+        /// Returns true if <paramref name="item"/> is a better match than <paramref name="other"/>.
```

Comment:
Well, it seems terribly fishy that an Is question is a tri-state, but at least documenting what all the returns means would be appreciated.